### ISSUE TITLE
Loading API description file does not work anymore in latest version

### ISSUE THREAD

**Issue by tiguchi** [stoicflame/enunciate#1031]
## Setup

- JAX-RS
- Gradle
- Enunciate Gradle Plugin version 2.13.0
- Enunciate XML configuration file that uses the `file` attribute of the `<description>` tag to load a Markdown text file

### Platform / OS

- OpenJDK 11
- Mac OS X 10.15.5

## Problem

The latest version of Enunciate fails to load a Markdown description file with the following error message:

```
/path/to/project/config/enunciate.xml/../doc/intro.md (Not a directory)
```

## Configuration File

```xml
<enunciate xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:noNamespaceSchemaLocation="http://enunciate.webcohesion.com/schemas/enunciate-2.13.0.xsd">

    <title>Test API</title>
    <description format="markdown" file="../doc/intro.md"/>
    <application root="https://api.example.com"/>

    <api-classes>
        <include pattern="com.example.**"/>
        <exclude pattern="org.jboss.**"/>
    </api-classes>

    <modules>
    </modules>
</enunciate>

```

## Demo Project

https://github.com/tiguchi/enunciate-description-file-bug

**Comment by stoicflame** [comment]
Thanks for the report and the demo project.

I'm not a gradle guy. Can you tell me how to build the demo project such that the problem is exposed? (I cloned the repo and ran `gradlew` and it built fine.)

**Comment by stoicflame** [comment]
Nevermind; I got it `gradlew enunciate`.

**Comment by stoicflame** [comment]
Fixed in 2.13.1.
